lib/inspec/resources/file.rb

```diff
@@ -40,6 +40,11 @@ def initialize(path)
       # select permissions style
       @perms_provider = select_file_perms_style(inspec.os)
       @file = inspec.backend.file(path)
+      validate_file_existence
+    end
+
+    def validate_file_existence
+      raise Inspec::Exceptions::ResourceFailed, "The file" + file.path + " doesn't seem to exist" unless exist?
```

Comment:
This change will break a lot of things. Most tests in inspec are designed to check separately for existence. I would remove this change.